webapp/utils/async_client.jsx

```diff
@@ -138,6 +138,36 @@ export function getMyChannelMembers() {
     });
 }
 
+export function getChannelMembersForUserIds(channelId, userIds, success, error) {
```

Comment:
Could you make this an action? Trying to use actions more, especially with success and error functions